.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       Django ${{ matrix.django-version }},
       Redis.py ${{ matrix.redis-version }}
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
why not using ubuntu-latest?

Reply:
Latest don't support Python 3.6.

I hit the issue here: https://github.com/jazzband/django-redis/actions/runs/3859407256/jobs/6578888882#step:3:7

Reply:
I think it can be 22.10, I'll try it. Latest is 23.04.

Reply:
already replied in [here](https://github.com/jazzband/django-redis/pull/644#issuecomment-1387068318)

Reply:
I tried 22.04 and it fails too.

I agree it could be setup with pyenv, I'll try it in another PR.